plugins/extractit/unit_test/test_url_extractor.py

```diff
@@ -163,3 +163,13 @@ def test_extract_urls_from_file_bad(self):
         )
         expected = {Output.URLS: []}
         self.assertEqual(actual, expected)
+
+    def test_extract_urls_from_html_email(self):
+        action = UrlExtractor()
+        actual = action.run(
+            {
+                Input.STR: '<html><head>\r\nhttps://www.example.com/phishing" style="box-sizing:border-box; line-height:inherit; margin:0px"><a href="https://www.example.com/extractit">https://www.example.com/extractit</a></p><p style="box-sizing:border-box; line-height:inherit; margin:0px">Normal text inlaid with my link</p></div></div></td></tr></tbody></table></th></tr></tbody></table></div></body></html>'
```

Comment:
I could be wrong but I think this might be missing some corresponding opening tags to some of the  corresponding closing tags at the end?

Reply:
You would be right- this used to be a lot longer but I cut it down significantly to reduce the amount of extra stuff in between tags (I kept the existing borders to links, but removed stuff in between these borders)

Reply:
Ideally we have correctly formatted HTML in this test, but we are only really concerned with finding URLs in it, so as long as those still match this is fine.